Makefile

```diff
@@ -25,8 +25,9 @@ BUILD_DIR ?= ./out
 ORG := github.com/GoogleCloudPlatform
 PROJECT := container-diff
 REPOPATH ?= $(ORG)/$(PROJECT)
+RELEASE_BUCKET ?= $(PROJECT)
```

Comment:
nit: can you default this to the release bucket we are using currently?

Reply:
PROJECT=container-diff

Reply:
that makes sense but I don't have access to a container-diff project.  Just wanted to make it clear where the releases are going